x-pack/test/observability_api_integration/trial/tests/annotations.ts

```diff
@@ -10,7 +10,7 @@ import { JsonObject } from '@kbn/utility-types';
 import { Annotation } from '@kbn/observability-plugin/common/annotations';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
 
-const DEFAULT_INDEX_NAME = 'observability-annotations';
+const DEFAULT_INDEX_NAME = 'observability-annotations-v1';
```

Comment:
```suggestion
const DEFAULT_INDEX_NAME = 'observability-annotations-v1';
```
This default should bring in the resource version constant to construct the version